tests/client/actions.test.js

```diff
@@ -0,0 +1,29 @@
+import {
```

Comment:
Can this file please be inside the `actions` folder and named accordingly?
Current path: `tests/client/actions.test.js`
Expected path: `tests/client/actions/categories.tests.js`